cfe_internal/enterprise/CFE_hub_specific.cf

```diff
@@ -548,7 +548,7 @@ bundle agent cfe_internal_clear_last_seen_hosts_logs
 
   commands:
 
-    (policy_server|am_policy_hub).enterprise_edition.active_hub::
+    (policy_server|am_policy_hub).enterprise_edition.active_hub.cfengine_3_12::
```

Comment:
On what versions do you expect the cli_task clearLastSeenHostsLogs will execute?

With this class expression, it will only execute if a host if running some version of 3.12 (e.g. 3.12.0, 3.12.3, 3.12.12) , it will no longer execute on hosts running 3.10 or 3.7.

Reply:
3.10 and 3.7 shouldn't execute this policy at all =) there is no cli task and lastseenhostlog table. I will check if all of 3.12.xx have to execute this policy

Reply:
yes, as I expected 3.12.0 has clearLastSeenHostsLogs CLI task, so, now this policy should be executed on all 3.12 hub 
versions

Reply:
Thanks, then this looks correct to me.